### PR TITLE
[Tests-Only]Configure expected failures file for remaining suites

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This PR Configure expected failures file for iPhone & XGA resolution

## Related Issue
- https://github.com/owncloud/web/issues/4850

## How Has This Been Tested?
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -170,6 +170,7 @@ config = {
 				]
 			},
 			'extraEnvironment': {
+			    'EXPECTED_FAILURES_FILE': '/var/www/owncloud/web/tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md',
```
Comment (individual-it) [inline .drone.star:173]: wrong indent

**Review by individual-it — CHANGES_REQUESTED** [review]
why do the line numbers need to be adjusted?

File: .drone.star (hunk 1 above)
Comment (jasson99) [reply .drone.star]: fixed

**Review by individual-it — APPROVED** [review]

File: .drone.star [hunk 2]
```diff
@@ -170,6 +170,7 @@ config = {
 				]
 			},
 			'extraEnvironment': {
+			        'EXPECTED_FAILURES_FILE': '/var/www/owncloud/web/tests/acceptance/tests/acceptance/expected-failures-XGA-with-oc10-server-oauth2-login.md',
```
Comment (dpakach) [inline .drone.star:173]:
```suggestion
			        'EXPECTED_FAILURES_FILE': '/var/www/owncloud/web/tests/acceptance/expected-failures-XGA-with-oc10-server-oauth2-login.md',
```

File: .drone.star [hunk 3]
```diff
@@ -231,6 +232,7 @@ config = {
 				]
 			},
 			'extraEnvironment': {
+			        'EXPECTED_FAILURES_FILE': '/var/www/owncloud/web/tests/acceptance/tests/acceptance/expected-failures-Iphone-oc10-server-oauth2-login.md',
```
Comment (dpakach) [inline .drone.star:235]:
```suggestion
			        'EXPECTED_FAILURES_FILE': '/var/www/owncloud/web/tests/acceptance/expected-failures-Iphone-oc10-server-oauth2-login.md',
```

File: .drone.star (hunk 2 above)
Comment (jasson99) [reply .drone.star]: fixed

File: .drone.star (hunk 3 above)
Comment (jasson99) [reply .drone.star]: fixed

**Comment by jasson99** [comment]
These changes are already merged through cherry-pick of the commit

**Pull request closed without merging**